app/src/main/java/com/kickstarter/models/Update.java

```diff
@@ -18,6 +18,7 @@
   public abstract @Nullable Integer commentsCount();
   public abstract @Nullable Boolean hasLiked();
   public abstract long id();
+  public abstract boolean isPublic();
```

Comment:
oh dang good catch. this will always be `false` since indeed, `public` rather than `is_public` is what's serialized. 

i missed this when looking at [our swift Update model]( https://github.com/kickstarter/ios-ksapi/blob/master/KsApi/models/Update.swift#L11) and neglecting the [actual decoding part](https://github.com/kickstarter/ios-ksapi/blob/master/KsApi/models/Update.swift#L46)